### ISSUE TITLE
New Nvidia drivers reducing performance?

### ISSUE THREAD

**Issue by TimmyP777** [Nukem9/dlssg-to-fsr3#358]
I was on 546.33 until yesterday. I thought it was the new FSR3 version of puredark, so I downgraded back to what I had before (this mod + DLSSFG build 02, im on version 10.0.0). I upgraded to 546.65 yesterday, and didn't play anything with this until a few minutes ago, Elden Ring. I have played dozens of hours with this.

To be clear: I DID redo the signature registry check.

Today, on 546.65 I immediately notice a difference, especially around Liurnia lakes where RT is heavier. I notice stutters. Downgrade? Problem fixed immediately, also redid registry to be clear.

Others NEED to look at this. It still worked, just not as well.

**Comment by Z4ndyz** [comment]
Have you taken shader recompilation into account if those shaders are built on the fly? That's usually what happens with DX12 games for a bit of time after updating the new driver

**Comment by TimmyP777** [comment]
Yes positive. Easy to reproduce.

Someone should try downgrading or something.

**Comment by MJ12MJ** [comment]
Why the hell are you bringing third party content to a GitHub page that has nothing to do with it and isn't even supported by this mod?
The Tab regarding issue is reserved for the mod not for Nvidia drivers or third party mods. Respect the author's work.

**Comment by TimmyP777** [comment]
Read the room dipshit

**Comment by TimmyP777** [comment]
I mean you're a fucking idiot. Nvidia very will could be dicks and ruin
this. Now thanks to you we have your stink in here.

**Comment by MJ12MJ** [comment]
This space is reserved for the mod, Nukem9 does not and cannot make changes at the driver level or fix stuff broken by that. Moderate your language and be more correct next time.

**Comment by TimmyP777** [comment]
No? I'm not going to do anything you tell me to do, and I laugh at the position you think you are in.

Nvidia could absolutely kill this in a driver and get away with it. They are selling a generation of cards with this feature as the main selling point.

It would be stupid, VERY stupid to not keep an eye on this potential issue.

Derr does Nukem9 work on Puredark mods? Because there are about 1000 other, VALID, posts you could "call out" Mr. Officer. Thanks for playing now gtfo.

**Comment by MJ12MJ** [comment]
Nvidia has every right to block the use of DLSSG with the AMD version as this mod allows it. If it had been a mod that implemented FSR3 without necessarily having to modify the Nvidia streamline then it would have been another matter, but since here we talk about manipulation of this streamline to replace DLSSG with the AMD variant expect that this could happen in the future, both engine level and especially at driver level. Next time report the mod issues not the Nvidia driver which causes you low fps which has nothing to do with what this mod offers.

"New Nvidia drivers reducing perfomance" 
Make a ticket to Nvidia o post a topic on their dedicated game ready drivers forum.

**Comment by TimmyP777** [comment]
Your on drugs or something...

No Im not gonna post this on Nvidia forums. Wtf?

**Comment by MJ12MJ** [comment]
> Youre on drugs or something...
> 
> No Im not gonna post this on Nvidia forums. Wtf?
> 
> *Like seriously get your brain checked. WOW.

Look at all your edits dude, there's a lot of stuff in there and I can see everything you wrote here.
Nvidia drivers have issue since 537.58 on some configurations and is already in progress to deliver a fix for it. So if you're experiencing low fps it's not because of this mod, of which I would like to remind you that it generates frames on the real basis that your GPU generates, so if you update the driver and find yourself with lower frames it is due to the driver and not the mod.

**Comment by MJ12MJ** [comment]
![Screenshot_20240120_183628](https://github.com/Nukem9/dlssg-to-fsr3/assets/154266015/3605eeb9-97bc-4234-b62e-36eedbfe1279)
Cleanse your brain and try to screw other people. You're ridiculous.

**Comment by TimmyP777** [comment]
Yeah... I deleted swears so you just got them in email, and I redacted my original claim which begs the question Dr. Science:

WHY ARE YOU EVEN POSTING? I made my redaction before you entered this thread.

Lol okay so post what I deleted. Let me be clear: I do not give a FUCK.

**Comment by MJ12MJ** [comment]
> Yeah... I deleted swears so you just got them in email, and I redacted my original claim which begs the question Dr. Science:
> 
> WHY ARE YOU EVEN POSTING? I made my redaction before you entered this thread.
> 
> Lol okay so post what I deleted. Let me be clear: I do not give a F*CK.
> 
> I mean every single word I say there. I wanna clarify that too. You are an IDIOT.

You haven't deleted anything, GitHub keeps track of the changes you make to your topics to avoid idiots like you and save time and resources for software developers.

GG, Well played and really smart!

**Comment by TimmyP777** [comment]
I edited THIS post, removing my issue. THEN you look at the ALREADY DELETED post, and respond as if that is current. You, are a sad, sad idiot. Holy sh*t.

**Comment by Z4ndyz** [comment]
This is no place for acting like this, this is you begging for an accout suspenssion , this is supposed to be a place to discuss issues, bugs, suggestions, contribute and code but i feel like this is a kindergarden, topic needs to be shut down there's nothing useful here

**Comment by TimmyP777** [comment]
I completely agree. He should not have responded to deleted material to create these issues!

**Comment by MJ12MJ** [comment]
> I edited THIS post, removing my issue. THEN you look at the ALREADY DELETED post, and respond as if that is current. You, are a sad, sad idiot. Holy sh*t.

You literally requested support for a mod that conflicts with another mod when the latter isn't even supported by the developer. You are absolutely ridiculous, if you have solved it, edit the topic and move it to the appropriate section. How old are you? 10? 😐

You've literally made 8+ edits to your topic asking for help with a software conflict from another mod that isn't compatible with this one. The one who uses drugs and probably even lives on the moon is you.

**Comment by TimmyP777** [comment]
What are you on? You posted my first post. If there are driver issues that cause performance loss in this mod, I sure as **** hope someone comes HERE and reports it.

Nothing you have done in this thread has made any sense, since I deleted the post you dug up for some reason? Why did you repost my original, deleted post? Its a non issue that I redacted?

**Comment by MJ12MJ** [comment]
This is precisely the reason, what the user requested here cannot be fulfilled for various reasons.

**Comment by TimmyP777** [comment]
Uh yes it can, when others have the same issue... Hence why I redacted it.

We arent going on Nvidia forums to talk about an issue like this. You are extremely misguided.

**Comment by MJ12MJ** [comment]
> Uh yes it can, when others have the same issue... Hence why I redacted it.
> 
> We arent going on Nvidia forums to talk about an issue like this. You are extremely misguided.
> 
> *To be clear, and since we are traveling back in time: I posted my issue to see if others could reproduce it, when I realized it was on MY END, I DELETED my post.
> 
> Then you came here... what is going on?

Your changes are visible to all users here on the platform, as I have already told you, this is precisely to avoid wasting time from people like you. You said you solved it, move your topic to the solved section and that's it. But I doubt you will since you're only here to report issues related to third-party software like the aforementioned puredark mod.

Nukem has already said that this mod is not compatible with other mods and that the problems inherent to its mod must be reported here and not the conflict between it and the puredark mod. Speak for yourself and not for others, since everyone here respects this very simple rule while you persist in denigrating others with insults and other things because your combination of mods doesn't work as you would like.

**Comment by TimmyP777** [comment]
WHAT?

There are 1000 posts here about using this MOD in conjunction with the OTHER Puredark MOD.

YOU are the one who looked at deleted, outdated material for some reason and responded to it. Any admin here can see that. YOU are the one propagating something that does not exist!

**Comment by MJ12MJ** [comment]
> WHAT?
> 
> There are 1000 posts here about using this MOD in conjunction with the OTHER Puredark MOD.
> 
> YOU are the one who looked at deleted, outdated material for some reason and responded to it. Any admin here can see that. YOU are the one propagating something that does not exist!
> 
> *Learn where you are "Read the room" before you speak. That was my FIRST RESPONSE to you. You STILL haven't done that. You clearly have not even read this ridiculous issue forum. You came STRAIGHT HERE.
> 
> Hilarious how I was one of the original people that said and supported a discord or subreddit. This is just pointless, you cause problems. Period.

hahahaha how long have you been using GitHub? 2 days? everyone can see what you do and what you edit. I still see that your topic is open even though you admitted to having solved the problem (and yet you didn't even close it), so from this I have every right to respond regardless of whether you like this action or not. And my response was simply that you cannot demand or get support for a mod combination (nukem + puredark) when the author himself does not support such compatibility. As for the nvidia drivers it is only necessary that the version you use supports DLSSG and that's it, if you have had framerate problems it is not caused by the nukem mod but by other factors, and if you are so stubborn as to say that it may have done so, publish your debugging report and prove that what you declare is at least truthful. We are all good at coming in here and reporting thousands of things, but if we don't have a basis of truth to support our statements you will never be taken seriously by anyone. When you have closed your topic because you have solved the problem, as you declared, it will be moved to the appropriate section and users will stop replying to you.

**Comment by MJ12MJ** [comment]
Let's put some other shit on nukem work, huh?
![Screenshot_20240120_191129_org mozilla firefox_edit_9307195454829](https://github.com/Nukem9/dlssg-to-fsr3/assets/154266015/afbbe078-e27d-44fa-8ce3-784f1d7f7d3f)
Shame ln you dude.

**Comment by TimmyP777** [comment]
AGAIN. READ THE ROOM. Seriously, this is no joke. You are totally, completely out of line here.

Let me clarify this for you:

People were using this mod, which is meant for ADDING frame generation via FSR3, to RTX cards, while retaining DLSS upscaling. That latter part is the basis for this mods existence; retaining DLSS upscaling.

Now, since Puredark ADDED NATIVE FSR3 FG, like what we have here, to his mods for frame generation... there is no need to use this mod in conjunction with his... So that post, was actually MINIMIZING posts we get here about the paid Puredark mods, since using his mod alone, only in the past few days, is a superior option to using this mod as it has UI fixes.

Its not shitting on his work, Dr. Science. It is deflecting Puredark users from coming here, as there is no longer a literal need.

Ive been online since it existence. I have met a ton of people. All mentalities, all walks of life... You are persisting in being the most lost person Ive ever spoken to. Everything you post is CLEARLY looking to incite some sort of problem.

**Comment by MJ12MJ** [comment]
> AGAIN. READ THE ROOM. Seriously, this is no joke. You are totally, completely out of line here.
> 
> Let me clarify this for you:
> 
> People were using this mod, which is meant for ADDING frame generation via FSR3, to RTX cards, while retaining DLSS upscaling. That latter part is the basis for this mods existence; retaining DLSS upscaling.
> 
> Now, since Puredark ADDED NATIVE FSR3 FG, like what we have here, to his mods for frame generation (which ALSO have decoupled DLSS upscaling)... there is no need to use this mod in conjunction with his... So that post, was actually MINIMIZING posts we get here about the paid Puredark mods, since using his mod alone, only in the past few days, is a superior option to using BOTH mods as it has UI fixes.
> 
> Its not shitting on his work, Dr. Science. It is deflecting Puredark users from coming here, as there is no longer a literal need.
> 
> Ive been online since it existence. I have met a ton of people. All mentalities, all walks of life... You are persisting in being the most lost person Ive ever spoken to. Everything you post is CLEARLY looking to incite some sort of problem.
> 
> For others: Puredark is slowly updating his old mods to include both forms of Frame generation. Not all have it yet, but Elden Ring and RDR2 do, and they work great with perfect UIs. No need for two mods anymore.
> 
> Do you get it, yet? The only proper response is silence.

You are a liar dude, if you think you did a great job you are dead wrong. Stop posting updates on mods that don't belong to nukem, you still don't understand this? you are absolutely ridiculous and sad.

**Comment by MJ12MJ** [comment]
Like other individuals who have created Patreons to sell his mod. You are all ridiculous, equally you who publish information on third party mods that do not belong to nukem, and you yourself report using it seeing the information in your comments. You're always trying to find an excuse to hide, but you're more ridiculous than I thought.

**Comment by TimmyP777** [comment]
Dude. Get lost. Seriously. Get help then come back. You have absolutely no business being here, and Im sure a dozen other places as well.

You are literally out of your league not only beyond anyone I have ever seen, but more than I thought possible.

The only explanation for your posts is raw ignorance, or intentionally looking to incite the maliciousness you "white knight" for. Nobody, but you, has ever posted anything like this.

...except me (pfff sigh), when I said we need to move these issues off of github... a month ago, multiple times. To no avail. Now that time has passed it really doesn't look like they minded, this is why you need to leave or participate.

So ridiculous whats happening here.

**Comment by alex-enginity** [comment]
Aside from insults and discussion.  I wanted to come here and confirm, the latest drivers completely destroyed my performance while doing Frame Generation.  I mean on Hogwarts it looks like it's doing it but it is actually rendering at 40fps or less while MSI afterburner says I'm at 118.

Aside from this mod, there is a performance issue in this driver, no doubt.  Without FG it still stutters like crazy. I'm talking about the 546.65.

I'm going back to 537

**Comment by alex-enginity** [comment]
Wanna know something more?.  The latest drivers are even bad at basic desktop.
Just moving a window on the desktop is a stuttering mess.  I can see all the contents on the window stuttering.  Going back to 546.17 and everything is super smooth again.

Swear to god NVIDIA keeps doing worst drivers every time.

**Comment by TimmyP777** [comment]
Yup. They are ****ing with it. 100 percent. That guy above is gaslighting; most likely an Nvidia employee.

RDR2 and W3 both had it disabled upon new driver reinstallation, it was enabled every start on .33 and smooth, and I could turn it on and off at will.

Now, on 465.65, it seems to be random. Witcher 3 turns off the FG option each time you start the game. Out of the 3 times I tried it, only 1 time worked (it worked for sure).

If they take this, I am done with this obvious money sink of a hobby after 40 years. They are taking advantage of the uninitiated.

Consider this: If you were\are on .65, you might not even be seeing the actual benefits this brings. It still works, but its clearly acting up.

**Comment by MJ12MJ** [comment]
> Yup. They are ****ing with it. 100 percent. That guy above is gaslighting; most likely an Nvidia employee.
> 
> RDR2 and W3 both had it disabled upon new driver reinstallation, it was enabled every start on .33 and smooth, and I could turn it on and off at will.
> 
> Now, on 465.65, it seems to be random. Witcher 3 turns off the FG option each time you start the game (it does not do this on 465.33). Out of the 3 times I tried it, only 1 time worked (it worked for sure).
> 
> If they take this, I am done with this obvious money sink of a hobby after 40 years. They are taking advantage of the uninitiated.
> 
> Consider this: If you were\are on 465.65, you might not even be seeing the actual benefits this brings. It still works, but its clearly acting up. Try 465.33 it works perfect all the time.
> 
> Cyberpunk also. Reinstalled 465.33 on top, immediately smooth again.
> 
> Now this thread makes more sense. Thats an Nvidia employee above, or a disgruntled 4070 upgrade.
> 
> THEY ARE BREAKING THIS!
> 
> *Last edit on this post:
> 
> Nvidia will slowly with each driver iteration, chip away at performance and usability of this. They cannot make a sudden change, because then its a "for sure" thing, and they destroy their "good will." So they first make it act "random" like this. This is the way Nvidia operates.
> 
> Others need to test and retest these claims, or we will be stuck on old driver versions. AMD should take note that this is messing with native FSR3 as well.

They are not breaking anything dude, Avatar offers a native implementation of FSR3 FG and there's literally no difference with drivers version from 545 to the actual one.
You're fucking dreaming or you're messing something with your system, because if what you're talking about was really big people's would obliterate Nvidia for this, even in games where developers supports FSR3 and not DLSS FG, because even 40 series users are locked with using it if devs doesn't add DLSS 3 natively.
Problems in the current driver branch exist and are related to the use of VSYNC and in combination with overlays for tracking GPU performance or hardware in general (such as HWINFO and Afterburner for example), frame generation does not has nothing to do with any of this and if you are getting low framerates with it it is because your base GPU is generating low fps, frame generation does nothing more than interpolate additional frames to those already present, but if you are having base issues with the driver making your GPU generate low fps it is obvious that the result obtained will not be the best. The only thing you can do at the moment is to do a general cleanup with DDU and stop using combinations of mods that are not compatible with each other (like puredark's mod with nukem's). Otherwise go back to driver 537.58 as recommended by Nvidia until they fix the problem. Simpler than that instead of acting like a conspiracy theorist.

**Comment by TimmyP777** [comment]
NOPE

**Comment by alex-enginity** [comment]
> > Yup. They are ****ing with it. 100 percent. That guy above is gaslighting; most likely an Nvidia employee.
> > RDR2 and W3 both had it disabled upon new driver reinstallation, it was enabled every start on .33 and smooth, and I could turn it on and off at will.
> > Now, on 465.65, it seems to be random. Witcher 3 turns off the FG option each time you start the game (it does not do this on 465.33). Out of the 3 times I tried it, only 1 time worked (it worked for sure).
> > If they take this, I am done with this obvious money sink of a hobby after 40 years. They are taking advantage of the uninitiated.
> > Consider this: If you were\are on 465.65, you might not even be seeing the actual benefits this brings. It still works, but its clearly acting up. Try 465.33 it works perfect all the time.
> > Cyberpunk also. Reinstalled 465.33 on top, immediately smooth again.
> > Now this thread makes more sense. Thats an Nvidia employee above, or a disgruntled 4070 upgrade.
> > THEY ARE BREAKING THIS!
> > *Last edit on this post:
> > Nvidia will slowly with each driver iteration, chip away at performance and usability of this. They cannot make a sudden change, because then its a "for sure" thing, and they destroy their "good will." So they first make it act "random" like this. This is the way Nvidia operates.
> > Others need to test and retest these claims, or we will be stuck on old driver versions. AMD should take note that this is messing with native FSR3 as well.
> 
> They are not breaking anything dude, Avatar offers a native implementation of FSR3 FG and there's literally no difference with drivers version from 545 to the actual one. You're fucking dreaming or you're messing something with your system, because if what you're talking about was really big people's would obliterate Nvidia for this, even in games where developers supports FSR3 and not DLSS FG, because even 40 series users are locked with using it if devs doesn't add DLSS 3 natively. Problems in the current driver branch exist and are related to the use of VSYNC and in combination with overlays for tracking GPU performance or hardware in general (such as HWINFO and Afterburner for example), frame generation does not has nothing to do with any of this and if you are getting low framerates with it it is because your base GPU is generating low fps, frame generation does nothing more than interpolate additional frames to those already present, but if you are having base issues with the driver making your GPU generate low fps it is obvious that the result obtained will not be the best. The only thing you can do at the moment is to do a general cleanup with DDU and stop using combinations of mods that are not compatible with each other (like puredark's mod with nukem's). Otherwise go back to driver 537.58 as recommended by Nvidia until they fix the problem. Simpler than that instead of acting like a conspiracy theorist.

I don't want to get into the discusssion.  Just to point that 465.65 are a complete disaster even WITHOUT frame generation.  The frame pacing is all over the place.

**Comment by TimmyP777** [comment]
> I don't want to get into the discusssion. Just to point that 465.65 are a complete disaster even WITHOUT frame generation. The frame pacing is all over the place.

Hmm. Did you ever wonder WHY that might be? They are the same branch with version numbers as close as any other driver version has been... Yet this is broken, because of some other issue?

Sorry there is nothing you can "get into" with me. I will keep this right here, watching it diligently regardless of whether this is explicitly happening now or not... I KNOW Nvidia. Been doing this a long time.

Its cute though... that you think you can even "get into it" with me. The only way you can really do so, is to work for Nvidia...

**Comment by alex-enginity** [comment]
When I say "I don't want to get into..."  i was referring that I am currently too busy working so I don't usually get into discussions. Some of your findings could be corroborated by me.  That's all I'm saying.
And yes, drivers 465.65 have horrible frame pacing issues.  Even going down 3 version below with the same 465 are like night and day difference.
